packages/core/addon/models/bard-request-v2/request.ts

```diff
@@ -241,35 +247,25 @@ export default class RequestFragment extends Fragment.extend(Validations) implem
     return this.columns.pushObject(this.fragmentFactory.createColumn(type, source, field, parameters, alias));
   }
 
-  /**
-   * Adds a new column with the parameters
-   */
-  addColumnFromMeta(columnMetadataModel: ColumnMetadataModels, parameters: Parameters): ColumnFragment {
-    return this.columns.pushObject(this.fragmentFactory.createColumnFromMeta(columnMetadataModel, parameters));
-  }
-
   /**
    * Adds a column with it's default parameters
    */
   addColumnFromMetaWithParams(columnMetadataModel: ColumnMetadataModels, parameters: Parameters = {}): ColumnFragment {
```

Comment:
should we rename this to `addColumnFromMeta`?

Reply:
Yea I think once we expect all tests to be passing we can start with some renaming, but maybe skip it for now